### ISSUE TITLE
Travis failing, latest version not in CI

### ISSUE THREAD

**Issue by benoror** [ride/ember-stripe-service#37]
Hola @buritica! :smile: 

I found your this stripe wrapper through [emberobserver](https://emberobserver.com/?query=stripe)

I see that the latest build version `4.0.0` is failing: https://travis-ci.org/ride/ember-stripe-service/builds

I can see it's failing with recent versions of Ember only: https://travis-ci.org/ride/ember-stripe-service/builds/71610063

Nevertheless I see the latests version you pushed `4.1.0`, although no build was triggered.

Is the build fixed in that version? Is it possible if you trigger latest build, or is there a way I can help you?

Thanks in advance! Cheers! :smile_cat:

**Comment by buritica** [comment]
Hey Ben,

Yeah I think I never got to see why it was breaking, cause tests run well locally. Seems to be something with Ember.view.extend:

https://travis-ci.org/ride/ember-stripe-service/jobs/71610065

If you wanna give it a look and have some suggestions on how to fix this they're welcome!

**Comment by benoror** [comment]
Afaik `Ember.View`'s are deprecated in Ember 2+, that might be the issue

**Comment by buritica** [comment]
Right, we don't use views so this is probably on the testing end and not the lib end. perhaps updates that come in ember-cli?

**Comment by benoror** [comment]
I see, maybe a Travis configuration?

**Comment by buritica** [comment]
do you have a fork and does it break too?

**Comment by benoror** [comment]
@luissifu can you clone it and run tests locally?

**Comment by benoror** [comment]
@buritica We ran the tests locally and had the same result as in https://travis-ci.org/ride/ember-stripe-service/jobs/71610064: 

```
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated
```

Maybe the offender is some outdated dependency (which also includes deprecated stuff like `Ember.View`)?

**Comment by buritica** [comment]
So, I think it can be the `STRIPE_PUBLISHABLE_KEY? or another .env var? there should def be some output or failures of some sort

https://github.com/ride/ember-stripe-service#running-tests

**Comment by buritica** [comment]
This is fixed, thanks to @ryanto for #38

**Comment by benoror** [comment]
Wow thanks @ryanto !!
